src/main/groovy/net/wasdev/wlp/gradle/plugins/extensions/LibertyExtension.groovy

```diff
@@ -20,10 +20,19 @@ import org.gradle.util.ConfigureUtil
 class LibertyExtension {
     
     String installDir
-    String outputDir
     String userDir
     String serverName = "defaultServer"
-
+    String outputDir
```

Comment:
This is a nit, but move the outputDir variable back up where it was after installDir